downstream/modules/platform/ref-controller-intro-job-template.adoc

```diff
@@ -44,7 +44,7 @@ For instructions, see the xref:controller-workflow-visualizer[Workflow Visualize
 
 == Domains
 
-You can filter templates by *Domain*. Specify a domain to make relevant resources easily accessible. Click the image:wrench.png[Configure domain, 15,15] icon to configure metadata for your domains, to add additional domains, or to image:delete-button[Delete,15,15] a domain.
+You can filter templates by *Domain*. Specify a domain to make relevant resources easily accessible. Click the image:wrench.png[Configure domain, 15,15] icon to configure metadata for your domains, to add additional domains, or to image:delete-button.png[Delete,15,15] a domain.
```

Comment:
The rich diff shows "DeleteDelete" and I'm not sure if it publishes that way? I don't know if there is a coding error there.